routers/api/v1/repo/issue_label.go

```diff
@@ -351,8 +351,8 @@ func prepareForReplaceOrAdd(ctx *context.APIContext, form api.IssueLabelsOption)
 	}
 
 	if !ctx.Repo.CanWriteIssuesOrPulls(issue.IsPull) {
```

Comment:
The permission check is a bit strange. Why pull request unit permission is checked?